### PR TITLE
Tibil develop 9oct

### PR DESCRIPTION
**Pull request by aks30**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -79,7 +82,7 @@ server.listen(port, function () {
 });
 server.on('error', onError);
 server.on('listening', onListening);
-
+server.setTimeout(500000);
```
Comment (aks30) [inline app.js]: @sharmashbee , @shrawanTibil - Is this required?

File: config/config.json.sample [hunk 2]
```diff
@@ -5,7 +5,26 @@
    "node_env": "local",
    "apidoc_url": "/apidoc",
    "apidoc_path":"/public/apidoc",
-
+   "file_path": {
```
Comment (aks30) [inline config/config.json.sample:8]: @sharmashbee , @shrawanTibil - Please provide values which can copy-paste from here to config.json

File: controllers/v1/portal_api.js [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+*     {
+*       "result": true,
+*       "data": [{
+        }]
+*     }
+   * @apiUse errorBody
+   */
+  exports.genericApi = async function (req, res) {
+    try {
+        let randomNumer = Math.random().toFixed(2)*100
```
Comment (aks30) [inline controllers/v1/portal_api.js]: @sharmashbee , @shrawanTibil - Please remove all debug related changes from all the concerend files

**Comment by sharmashbee** [comment]
Hi Akash,
I made the changes which you have mentioned and now latest code i had
pushed.
Thanks
sharmashbee

**Pull request merged**